src/en/about/guide.md

```diff
@@ -1,57 +1,95 @@
 
-All admin actions on communties are done on the community sidebar. This currently only includes Removing / Restoring communities.
+## Using Markdown
 
-## Markdown Guide
+Lemmy's posts are done using Markdown, there are of course shortcuts for you to use when writing but here is a table to help you if you want to go further.
 
-Type | Or | … to Get 
---- | --- | ---
-\*Italic\* | \_Italic\_  | _Italic_ 
-\*\*Bold\*\* | \_\_Bold\_\_ | **Bold** 
-\# Heading 1 | Heading 1 <br> ========= | <h4>Heading 1</h4>
-\## Heading 2 | Heading 2 <br>--------- | <h5>Heading 2</h5>
-\[Link\](http://a.com) | \[Link\]\[1\]<br>⋮ <br>\[1\]: http://b.org | [Link](https://commonmark.org/) 
-!\[Image\](http://url/a.png) | !\[Image\]\[1\]<br>⋮ <br>\[1\]: http://url/b.jpg | ![Markdown](https://commonmark.org/help/images/favicon.png) 
-\> Blockquote | | <blockquote>Blockquote</blockquote>
-\* List <br>\* List <br>\* List | \- List <br>\- List <br>\- List <br> | *   List <br>*   List <br>*   List <br>
-1\. One <br>2\. Two <br>3\. Three | 1) One<br>2) Two<br>3) Three | 1.  One<br>2.  Two<br>3.  Three
-Horizontal Rule <br>\--- | Horizontal Rule<br>\*\*\* | Horizontal Rule  <br><hr>
-\`Inline code\` with backticks | |`Inline code` with backticks 
-\`\`\`<br>\# code block <br>print '3 backticks or'<br>print 'indent 4 spaces' <br>\`\`\` | ····\# code block<br>····print '3 backticks or'<br>····print 'indent 4 spaces' | \# code block <br>print '3 backticks or'<br>print 'indent 4 spaces'
-::: spoiler hidden or nsfw stuff<br>*a bunch of spoilers here*<br>::: | | <details><summary> hidden or nsfw stuff </summary><p><em>a bunch of spoilers here</em></p></details>
-Some ~subscript~ text | | Some <sub>subscript</sub> text
-Some ^superscript^ text | | Some <sup>superscript</sup> text
+| Type | Or  | … to Get |
+| --- | --- | --- |
+| \*Italic\* | \_Italic\_ | *Italic* |
+| \*\*Bold\*\* | \_\_Bold\_\_ | **Bold** |
+| \# Heading 1 | Heading 1<br>========= | #### Heading 1 |
+| \## Heading 2 | Heading 2<br>\-\-\-\-\-\-\-\-\- | ##### Heading 2 |
+| \[Link\](http://a.com) | \[Link\]\[1\]<br>⋮<br>\[1\]: http://b.org | [Link](https://commonmark.org/) |
+| !\[Image\](http://url/a.png) | !\[Image\]\[1\]<br>⋮<br>\[1\]: http://url/b.jpg | ![Markdown](https://commonmark.org/help/images/favicon.png) |
+| \> Blockquote |     | > Blockquote |
+| \* List<br>\* List<br>\* List | \- List<br>\- List<br>\- List | \* List<br>\* List<br>\* List |
+| 1\. One<br>2\. Two<br>3\. Three | 1) One<br>2) Two<br>3) Three | 1\. One<br>2\. Two<br>3\. Three |
+| Horizontal Rule<br>\-\-\- | Horizontal Rule<br>\*\*\* | Horizontal Rule<br><br>* * * |
+| \`Inline code\` with backticks |     | `Inline code` with backticks |
+| ```<br>\# code block<br>print '3 backticks or'<br>print 'indent 4 spaces'<br>``` | ····# code block<br>····print '3 backticks or'<br>····print 'indent 4 spaces' | \# code block<br>print '3 backticks or'<br>print 'indent 4 spaces' |
```

Comment:
This seemed to break. If there's no content changes, I think I'd prefer it the old way. I didn't load this up in my markdown parser, but on github some of this looks broken.